js/components/items-container.tsx

```diff
@@ -386,12 +375,8 @@ class ItemsContainer extends React.Component<IComponentProps, IItemsContainerSta
 
         } else if (model.objectDataRequest || model.fileDataRequest) {
             objectData = model.objectData;
-            limit = parseInt(
-                manywho.settings.flow(
```

Comment:
It didn't use to apply pagination attributes here, so I want to be super sure that we should.
Looking at the docs https://github.com/manywho/ui-html5/wiki/Table-Component seems to suggest the pagination attribute only applies to List Values, but then the paginationSize is hinted to also apply to Service ObjectDataRequests.

Reply:
Okay, so re-reading those docs it seems the "pagination" attribute should only apply to list values to turn pagination on/off. 
The "paginationSize" attribute should set the page size for all Service ObjectDataRequests and only list values where the "pagination" attribute has been set to true.

Reply:
Now handling this in the getPageSize function.